internal/utils/browser.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"github.com/pkg/browser"
+)
+
+// BrowserInterface provides access to browser
+type BrowserInterface interface {
+	Open(url string) error
+}
+
+// Browser is a mocked browser
+type Browser struct{}
+
+// Open opens the default browser.
+func (*Browser) Open(url string) error {
+	return browser.OpenURL(url)
```

Comment:
[question] I need to look into this package more closely myself, but are there any options/configurations to provide when calling `OpenURL`?  if, at the end of the day, we're just leveraging a package's static function call, having this be it's own "thing" (with interface and all) may be overkill

Reply:
this is the package if you're interested in checking it out: https://github.com/pkg/browser
to my knowledge it's a very basic package that doesn't offer options/configurations